### ISSUE TITLE
Debian image fails to boot with LXD installed

### ISSUE THREAD

**Issue by leonklingele** [ayufan-rock64/linux-build#337]
The latest Debian image (`0.8.0rc5: gitlab-ci-linux-build-107`) fails to boot when LXD is installed (same issue with latest stable Debian image). Unfortunately, I don't have a working UART adapter at hand, so I can't debug this myself :(

Steps to reproduce:

```sh
$ apt update
$ apt install -y --no-install-recommends snapd
$ snap install lxd
$ yes '' | /snap/bin/lxd init
$ reboot
# Board will never come back
```

**Comment by ayufan** [comment]
Will check :)

**Comment by leonklingele** [comment]
Reproducible with Bionic as well when LXD is installed as a snap. Works fine when using the LXD deb package.

**Comment by Halytskyi** [comment]
Seems it's related to issue: https://github.com/ayufan-rock64/linux-build/issues/332

**Comment by Halytskyi** [comment]
Any chance for fix that? Tried the last kernel for now (4.4.167-1189-rockchip-ayufan-gea9ef7a80268) still the same issue...

**Comment by Halytskyi** [comment]
Seems I found workaround (as temporary solution), can be useful for someone until it won't be fixed on kernel level. Need start snap-core.mount (and all snapd stuff) at the end of system startup.

Verify if installed only one "snap-core" (as mount will be enabled for all core's):
```
# systemctl list-unit-files | grep snap-core | grep mount
snap-core-6410.mount                         enabled
snap-core-6533.mount                         enabled
snap-core-6962.mount                         enabled

# snap list --all
Name        Version                Rev    Tracking  Publisher            Notes
core        16-2.37.2              6410   stable    canonical✓           core,disabled
core        16-2.37.4              6533   stable    canonical✓           core,disabled
core        16-2.39                6962   stable    canonical✓           core
```

In my case were installed also old versions. Uninstalled them by command:
```
# snap remove core --revision=6410
core (revision 6410) removed

# snap remove core --revision=6533
core (revision 6533) removed
```

Also, to avoid auto-upgrade LXD cluster (and snapd) I disabled and stopped snapd service:
```
systemctl stop snapd.service
systemctl disable snapd.service
systemctl mask snapd.service
```

**Our temporary "fix"**
Disable next services:
```
systemctl disable snap-core-6962.mount
systemctl disable snap.lxd.activate.service
```

Add them to '/etc/rc.local' before 'exit 0' for start after system UP:
```
systemctl start snap-core-6962.mount
systemctl start snap.lxd.activate.service
```

After that my boards reboot without any issues.

P.S. Will be enough disable autostart for "snap-core-6962.mount" but in that case for start LXD you need manually start this mount after board started.

**Comment by leonklingele** [comment]
Apparently fixed with latest Debian Buster image (`0.9.0: gitlab-ci-linux-build-142`). Will give it a few more days until I will close this.

@ayufan can you please release a stable version as soon as possible? Thank you so much!

**Comment by ayufan** [comment]
The 0.9.x will not be stable for another 2-3 months.
